ConsoleGame.java

```diff
@@ -0,0 +1,46 @@
+public class ConsoleGame {
+    private final Game game;
+
+    public ConsoleGame(Game game) {
+        this.game = game;
+    }
+
+    public String drawBoard(Board board) {
+        print("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|");
+        return String.format("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|");
+    }
+
+    public void drawTurn() {
+        print("-----------------");
+        print("Its your turn " + game.playerActive().getMark() + "!");
+        print("-----------------");
+    }
+
+    public void print(String text) {
+        System.out.println(text);
```

Comment:
Output stream is a dependency so it should be injected into ConsoleGame class. Don't use System.out for Console display, use OutputStream or a child of..